### ISSUE TITLE
How do we deal with past exhibitions

### ISSUE THREAD

**Issue by jennpb** [wellcometrust/wellcomecollection.org#2070]
## Type
:speech_balloon: Team conversation

- Role of 'past' exhibitions in overall experience
- How do users navigate to past things?
- What happens on list pages when an exhibition is in the past?

Currently, CGDSYL is still listed on https://wellcomecollection.org/exhibitions even though it is now closed. This will be very confusing to people.

**Comment by jennpb** [comment]
Relates to #1916 (migrate useful past exhibitions) and #1745 (site audit)

**Comment by jennpb** [comment]
Content team plan:
- We used Tibet as a test case, using Ben's selected image and Bernadette's suggested copy in Trello. 
- We used the migrated version as James is confident that he can make it work.
- We made the first sentence of B's copy the 'featured' text and the rest as normal text.
- Publishing worked - https://wellcomecollection.org/exhibitions/W3Ls-SkAACgAEWMx - but it's not findable by Google/otherwise until James finishes the redirect/reordering tomorrow. We have flagged with Kate for a proofread before that happens.
- We know it appears on the exhibitions index page but this not an issue as it's not a high traffic page. 
- James will let us know after once that's done done so we can see that it's redirecting from the old Tibet page to the new one.
- We've decided to go ahead with publishing the past exhibitions as we're ready -- they don't need to be done in bulk. So Bernadette, once you get your copy in, Ben, you get your copy in, and arrange a quick proofread it can be published. Given it didn't take too long, it's doable to get all of these published by the end of August.
- We discussed that longer term adding a review quote could help, even though the exhibition has closed as it raises the profile of why it was good. It would involve research though so we need to decide more generally what our 'phase 4' approach to exhibitions (which is after they've closed). For example, does anything related to access disappear from the page.  And how can we drive onward journeys (in the form of another physical visit option, a read option and a research option)?
